frontend/src/app/layout/layout.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="flex justify-start items-start gap-8">
-  <div class="bg-slate-700 w-16 h-screen">
-    sidebar
-    <button class="text-green-400" (click)="signOut()">logout</button>
+  <div class="bg-slate-700 w-1/12 h-screen grid-flow-col row-span-1">
+    <button class="text-green-400" (click)="signOut()">Log Out</button>
+    <button class="text-yellow-400" (click)="changePassword()">Change Password</button>
```

Comment:
z poprzedniego review:

https://angular.io/api/router/RouterLink#setting-and-handling-query-params-and-fragments

ponieważ `changePassword` robi tylko `navigateByUrl` to powinien być linkiem/anchor (`<a>`) a nie buttonem